front/components/agent_builder/capabilities/mcp/MCPServerViewsDialog.tsx

```diff
@@ -0,0 +1,694 @@
+import type { MultiPageDialogPage } from "@dust-tt/sparkle";
+import {
+  Button,
+  LightbulbIcon,
+  MultiPageDialog,
+  MultiPageDialogContent,
+  MultiPageDialogTrigger,
+  Spinner,
+} from "@dust-tt/sparkle";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { uniqueId } from "lodash";
```

Comment:
Better to import the specific function to avoid bringing all of lodash.
```suggestion
import uniqueId from "lodash/uniqueId";
```